### PR TITLE
Add docs for recent features

### LINKED ISSUE #418: Update azure.json example
for new features such as multiple slb and tags.

### LINKED ISSUE #419: Add docs for recent features
- [x] multiple slbs
- [x] tags
- [x] custom health probes

### PR DESCRIPTION
**Pull request by nilo19**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake
/kind documentation

**What this PR does / why we need it**:
Add docs for recent features:
1. custom health probe
2. multiple standard load balancer
3. custom tags

**Which issue(s) this PR fixes**:

Fixes #419 
Fixes #418 

**Special notes for your reviewer**:

**Release note**:
```
NONE
```

### CODE REVIEW

File: examples/azure.json [hunk 1]
```diff
@@ -27,5 +27,7 @@
     "useInstanceMetadata": true,
     "loadBalancerSku": "<loadbalancer-sku>",
     "excludeMasterFromStandardLB": false,
-    "maximumLoadBalancerRuleCount": 250
+    "maximumLoadBalancerRuleCount": 250,
+    "enableMultipleStandardLoadBalancers": false,
+    "tags": "a=b,c = d"
```
Comment (feiskyer) [inline examples/azure.json:32]: let's remove spaces in example

File: site/content/en/topics/custom-health-probe.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+type: docs
+---
+
+Currently, the protocol of the health probe is defined as below:
+
+1. for local services, HTTP and /healthz would be used;
+1. for cluster TCP services, TCP would be used;
+1. for cluster UDP services, no health probes.
+
+Here we introduce two service annotations `service.beta.kubernetes.io/azure-load-balancer-health-probe-protocol` and `service.beta.kubernetes.io/azure-load-balancer-health-probe-request-path`, which determine the new health probe behavior. If the `service.beta.kubernetes.io/azure-load-balancer-health-probe-protocol` is set, both local and cluster TCP services would use the specified health probe protocol. If the `service.beta.kubernetes.io/azure-load-balancer-health-probe-request-path` is set, the specified request path would be used instead of `/healthz`. Note that the request path would be ignored when using TCP or the `service.beta.kubernetes.io/azure-load-balancer-health-probe-protocol` is empty.
```
Comment (feiskyer) [inline site/content/en/topics/custom-health-probe.md:14]: s/Here we introduce two service annotations/Since v1.20, two service annotations are introduced

Comment (feiskyer) [inline site/content/en/topics/custom-health-probe.md:14]: Could you add a table on the LoadBalancer probe setup based on new annotations and externalTrafficPolicy?

File: site/content/en/topics/multiple-slb.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
-title: "Design proposal: Support multiple Standard LoadBalancer in one cluster"
-linkTitle: "Design proposal: Support multiple Standard LoadBalancer in one cluster"
-weight: 6
+title: "Support multiple Standard LoadBalancer in one cluster"
```
Comment (feiskyer) [inline site/content/en/topics/multiple-slb.md:2]: rename title to "Multiple Standard LoadBalancer in the same cluster"

File: examples/azure.json (hunk 1 above)
Comment (nilo19) [reply examples/azure.json:32]: I just want to show that both formats are good.

**Review by feiskyer — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**